### PR TITLE
Make farmer not wondering about explosion forever 

### PR DESCRIPTION
**Pull request by vladtepesch**
fixes https://github.com/diasurgical/devilutionX/issues/2590

### CODE REVIEW

File: Source/towners.cpp [hunk 1]
```diff
@@ -630,6 +630,9 @@ void TalkToFarmer(PlayerStruct &player, TownerStruct &farmer)
 		if (gbIsMultiplayer)
 			NetSendCmdQuest(true, Q_FARMER);
 		break;
+	case QUEST_HIVE_DONE:
+		// do nothing
+		return;
```
Comment (AJenbo) [inline Source/towners.cpp:635]:
```suggestion
		break;
```

Comment (StephenCWills) [reply Source/towners.cpp:635]: The only place in this whole file where `return` is used in a switch is the `TalkToGirl()` function. Personally, I would very much prefer `break` than `return` here.

Comment (vladtepesch) [reply Source/towners.cpp:635]: Oh - I mislooked here - I interpreted as adding a `break` after `return` (which is enforeced by some code styles) - Sorry for confusion. I see - reakng here would do the same

Comment (AJenbo) [reply Source/towners.cpp]: Sounds like a terribly redundant code style since it would be unreachable code :D

**Review by AJenbo — APPROVED** [review]

**Pull request merged**